### PR TITLE
Rename Attributes to Parameters in docstrings

### PR DESCRIPTION
**Pull request by StFroese**

### CODE REVIEW

File: ctapipe/atmosphere.py [hunk 1]
```diff
@@ -169,7 +169,7 @@ class ExponentialAtmosphereDensityProfile(AtmosphereDensityProfile):
         density_profile.peek()
 
 
-    Attributes
+    Parameters
```
Comment (maxnoe) [inline ctapipe/atmosphere.py:172]: But attributes is correct. It's about the class attributes. Parameters are function arguments

Comment (StFroese) [reply ctapipe/atmosphere.py:172]: you're totally right

Comment (StFroese) [reply ctapipe/atmosphere.py:172]: don't know why sphinx didn't want to build it on my machine

Comment (StFroese) [reply ctapipe/atmosphere.py:172]: I'll close this again

**Pull request closed without merging**